### PR TITLE
Updates to the benchmarking script

### PR DESCRIPTION
**Pull request by timwoj**
- Added --mode argument for setting the benchmarking mode
- Added "file" mode for reading a file directly from disk with the Zeek '-r' argument
- Added reverse flamegraphs to the output from the flamegraph mode
- Added --seed argument for passing a random seed file to Zeek

### CODE REVIEW

File: devel-tools/perf-benchmark [hunk 1]
```diff
@@ -54,54 +70,77 @@ while (( "$#" )); do
...
+ZEEK_ARGS=""
+if [ "${MODE}" != "file" ]; then
+    ZEEK_ARGS="-i af_packet::${INTERFACE}"
+fi
+
+if [ -n "${SEED_FILE}" ]; then
+    ZEEK_ARGS="${ZEEK_ARGS} -G ${SEED_FILE}"
+fi
+
+PRELOAD_LIBS=`for i in $(ls -1L /home/tim/3rdparty/libs/*); do echo -n "$(readlink -f $i) "; done`
```
Comment (jsiwek) [inline devel-tools/perf-benchmark]: This path-hardcoding probably not meant to be included?

Then also the test right below probably means to check for non-zero string length w/ `-n`.

Comment (timwoj) [reply devel-tools/perf-benchmark]: Aw damn, I meant to remove that before I PR'd and forgot it. I'll take it out.

Comment (timwoj) [reply devel-tools/perf-benchmark]: Fixed.
